packages/presentation/lib/screen/main_data.dart

```diff
@@ -37,7 +37,11 @@ class MainData {
         isDisabled: isDisabled,
       );
 
-  MainData copyWith({int? randomNumber, int? guessNumber, required int counter, bool? isDisabled}) =>
+  MainData copyWith(
+          {int? randomNumber,
+          int? guessNumber,
+          required int counter,
+          bool? isDisabled}) =>
```

Comment:
strange formatting. did u place params to new lines manually or studio did it?

Reply:
it is command+option+L

Reply:
comma after last param